index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8">
+    <title>Meteor-gazelle by jrherr</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
+  </head>
+  <body>
+    <section class="page-header">
+      <h1 class="project-name">Meteor-gazelle</h1>
+      <h2 class="project-tagline">A framework built using Meteor.</h2>
+      <a href="https://github.com/jrherr/meteor-gazelle" class="btn">View on GitHub</a>
+      <a href="https://github.com/jrherr/meteor-gazelle/zipball/master" class="btn">Download .zip</a>
+      <a href="https://github.com/jrherr/meteor-gazelle/tarball/master" class="btn">Download .tar.gz</a>
```

Comment:
I assume this was auto-generated, can you fix the links to point at the main repo instead of yours? :)